test/acceptance/random.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var Mocha = require('../..').Mocha;
+
+describe('random option', function() {
+  describe('seed', function() {
+    it('should return a number', function() {
+      expect(Mocha.seed())
+        .to
+        .be
+        .a('number');
+    });
+
+    it('should return a positive number', function() {
+      expect(Mocha.seed() > 0)
+        .to
+        .be(true);
+    });
```

Comment:
This will give the nondescript "expected true actual false" thing if it fails. `expect(Mocha.seed()).to.be.above(0)` will be more helpful.

However, the test doesn't match the description; 0 is a positive number. If 0 is a valid output, then [in lieu of a greaterThanOrEqual assertion](https://github.com/Automattic/expect.js/issues/108) we can use `.within(0, Infinity)`. If not, the description should be updated instead.

See: https://github.com/Automattic/expect.js#api